App.js

```diff
@@ -2,10 +2,13 @@ import React from 'react';
 import {StyleSheet} from 'react-native';
 
 import Tabs from "./src/navigation/Tabs";
+import Profile from "./src/components/Profile/Profile";
+
 export default function App() {
 
   return (
-        <Tabs/>
+        // <Tabs/>
```

Comment:
Your code in pull request is code which will go to prod. It should be clean and organized. Please do not leave comments in prod code.